### ISSUE TITLE
Unable to use Element Picker & Zapper

### ISSUE THREAD

**Issue by troysjanda** [uBlockOrigin/uBlock-issues#731]
### Prerequisites

- [x ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [x ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://old.reddit.com/r/uBlockOrigin/)
    
- [x ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [x ] uBlock Origin is the only extension
    - [x ] uBlock Origin with default lists/settings
    - [x ] using a new, unmodified browser profile
- [x ] I am running the latest version of uBlock Origin
- [x ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description
Seems that on Ghacks the element picker & zapper do not work
[Description of the bug or feature]
Go to Ghacks and try to select the zapper or element picker nothing will happen
### A specific URL where the issue occurs
ghacks.net
[A specific URL is MANDATORY for issue happening on a web page, even if it happens "everywhere"]

### Steps to Reproduce

1. [First Step] goto ghacks.net
2. [Second Step] try to use or select the zapper/element picker
3. [and so on...] the screen does not gray and unable to select any items

### Expected behavior:
able to pick and hind element
[What you expected to happen]

### Actual behavior:
selecting the zapper/element picker does nothing
[What actually happened]

### Your environment

* uBlock Origin version: 1.22.3b16
* Browser Name and version: Firefox 69.0.1
* Operating System and version: Windows 10 Enterprise 1909 18363.356
* personal filterlist `https://raw.githubusercontent.com/troysjanda/MyBlockLists/master/UBOXPI`
### Note
Does not seem to happen with  Edge Chrome, as I am able to use both the zapper and element picker

**Comment by gwarser** [comment]
Make sure cosmetic filtering is enabled.

**Comment by uBlock-user** [comment]
Run the logger and look if _Disable Cosmetic Filtering_ is activated on that domain, or if `$elemhide` filter is there ?

**Comment by troysjanda** [comment]
I resolved it

**Comment by troysjanda** [comment]
not the same issue as #3212 as cosmedic filtering was not disabled. Still having the issue can't seem to figure out why

**Comment by troysjanda** [comment]
I think that it maybe related to the speci

**Comment by gwarser** [comment]
In which build it worked for you?

**Comment by uBlock-user** [comment]
> not the same issue as #3212 as cosmedic filtering was not disabled. Still having the issue can't seem to figure out why

and yet you said you resolved it ?

**Comment by uBlock-user** [comment]
DId you run the logger and check for `$elemhide` filter on your side ?

**Comment by troysjanda** [comment]
@uBlock-user I thought that i resolved it. I looked in the logger and did not see `$elemhide` 

@gwarser it seemed to be fine with the 1.22.3b13

I don't know any other ways to troubleshoot this. I have set up a VM with a fresh install of Windows 10 18363.356, installed latest FF 69.0.1 with Just Ubo 1.22.3b16  and still having the issue. Everything seems to work with Ubo but Zapper and Element Picker. Only seems to work if I downgrade Ubo to 1.22.3b13 which seems to be the build before the new feature was introduced. I am at a loss and Ill just have to deal with it. I have not found any other sites but Ghack.net that is affected.

**Comment by uBlock-user** [comment]
> before the new feature

Which new feature ?

**Comment by troysjanda** [comment]
@uBlock-user 
Static filter option elemhide as per ABP semantic

The elemhide option is now fully supported, rather than being an alias of generichide. The elemhide option will be internally converted into two filters, generichide and specifichide. There have been cases raised by filter list maintainers where specifichide would be useful. Additionally, the filter options elemhide, generichide and specifichide can be aliased with ehide, ghide and shide respectively. (generichide appears over 1,300 times just in "uBlock filters".)

**Comment by gorhill** [comment]
> Just Ubo 1.22.3b16 and still having the issue

Does the browser console report anything?

**Comment by troysjanda** [comment]
here is the browser console
`https://i.postimg.cc/m20SdGT7/ghacks.png`

**Comment by uBlock-user** [comment]
What about extension console ? Any errors or warnings there ?

**Comment by troysjanda** [comment]
From the logger (sorry I could not fit the whole output in a screen grab) 
```
<details><summary>Logger output</summary>

| | | | | | | |
|:--- |:--- |:--- |:--- |:--- |:--- |:--- |
| +0 | `##[href="https://deals.ghacks.net/"]` |  | `www.ghacks.net` |  | dom | `https://www.ghacks.net/` |
| +0 | ##.ghacks_share_widget |  | `www.ghacks.net` |  | dom | `https://www.ghacks.net/` |
| +0 | ##.ghacks_ad_code |  | `www.ghacks.net` |  | dom | `https://www.ghacks.net/` |
| +0 | ##.ghacks-sidebar > [id^="ghacks_ad_code-"]:not(#ghacks_ad_code-6) |  | `www.ghacks.net` |  | dom | `https://www.ghacks.net/` |
| +0 | ##.ghacks-ad-loaded |  | `www.ghacks.net` |  | dom | `https://www.ghacks.net/` |
| +0 |  |  | behind-the-scene | 0,3 | xhr | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/plugins/all-in-one-schemaorg-rich-snippets/css/style-121665b0f8e6664ac87391e39b0f49af.css` |
| +0 |  |  | behind-the-scene | 0,3 | xhr | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/plugins/all-in-one-schemaorg-rich-snippets/css/jquery.rating-12662fe6960c1c20be7bc8df91d5791a.css` |
| +0 |  |  | behind-the-scene | 0,3 | xhr | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/themes/new-ghacks-preview/style-2a9dadc0afc4c848202166205fd4bd38.css` |
| +0 |  |  | behind-the-scene | 0,3 | xhr | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/plugins/download-monitor/assets/css/frontend-e1fa022cfb79197603a09b386068cd69.css` |
| +0 |  |  | behind-the-scene | 0,3 | xhr | `https://www.ghacks.net/wp-includes/css/dist/block-library/style.min.css?ver=5.2.3` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/favicon.ico` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/Meld-folder-comparison-tool-300x300.jpg` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/antennapod-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/windows-10-view-optional-updates-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/firefox-69.0.1-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/Metatogger-is-a-great-way-to-manage-your-music-library-300x300.jpg` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/vivaldi-2.8-sync-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/microsoft-font-cascadia-code-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/sale_19399_primary_image-300x300.jpg` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/How-to-create-a-private-torrent-using-qBittorrent-300x300.jpg` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2017/09/microsoft-365-300x247.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/chrome-back-forward-cache-improvement-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/How-to-whiteout-text-in-PDFs-using-DocHub-300x300.jpg` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/opatch-windows7-support-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/microsoft-edge-chromium-pua-protection-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/devicecensus-exe-process-windows-10-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2018/01/windows-10-display-telemetry-data-300x300.png` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/Undo-Close-Tab-is-an-add-on-that-displays-a-clickable-list-of-recently-closed-tabs-in-Firefox-300x300.jpg` |
| +0 |  |  | `www.ghacks.net` | 1 | image | `https://www.ghacks.net/wp-content/uploads/2019/09/opera-mini-offline-file-share-300x300.png` |
| +0 | `\|\|www.googletagmanager.com^` | -- | `www.ghacks.net` | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-111230-1` |
| +0 | \|\|h-bid.com^$3p | -- | `www.ghacks.net` | 3 | script | `https://static.h-bid.com/ghacks.net/20190704/snhb-ghacks.net.min.js?20190704` |
| +0 |  |  | `www.ghacks.net` | 1 | css | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/plugins/all-in-one-schemaorg-rich-snippets/css/style-121665b0f8e6664ac87391e39b0f49af.css` |
| +0 |  |  | `www.ghacks.net` | 1 | css | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/plugins/all-in-one-schemaorg-rich-snippets/css/jquery.rating-12662fe6960c1c20be7bc8df91d5791a.css` |
| +0 |  |  | `www.ghacks.net` | 1 | css | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/themes/new-ghacks-preview/style-2a9dadc0afc4c848202166205fd4bd38.css` |
| +0 |  |  | `www.ghacks.net` | 1 | css | `https://www.ghacks.net/wp-content/cache/min/1/wp-content/plugins/download-monitor/assets/css/frontend-e1fa022cfb79197603a09b386068cd69.css` |
| +0 |  |  | `www.ghacks.net` | 1 | css | `https://www.ghacks.net/wp-includes/css/dist/block-library/style.min.css?ver=5.2.3` |
| +0 | \|\|h-bid.com^$3p | -- | `www.ghacks.net` | 3 | script | `https://static.h-bid.com/sncmp/sncmp_stub.min.js` |
| +0 | \|\|googlesyndication.com^ | -- | `www.ghacks.net` | 3 | script | `https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js` |
| +0 |  |  | `www.ghacks.net` | 1 | script | `https://www.ghacks.net/wp-content/themes/new-ghacks-preview/scripts.js?ver=1.2.3.4.5` |
| +0 | `\|\|www.googletagmanager.com^` | -- | `www.ghacks.net` | 3 | script | `https://www.googletagmanager.com/gtag/js?id=UA-111230-1` |
| +0 | \|\|h-bid.com^$3p | -- | `www.ghacks.net` | 3 | script | `https://static.h-bid.com/ghacks.net/20190704/snhb-ghacks.net.min.js?20190704` |
| +0 | \|\|h-bid.com^$3p | -- | `www.ghacks.net` | 3 | script | `https://static.h-bid.com/sncmp/sncmp_stub.min.js` |
| +0 | \|\|googlesyndication.com^ | -- | `www.ghacks.net` | 3 | script | `https://pagead2.googlesyndication.com/pagead/js/adsbygoogle.js` |
| +0 |  |  | `www.ghacks.net` | 1 | script | `https://www.ghacks.net/wp-content/plugins/all-in-one-schemaorg-rich-snippets/js/jquery.rating.min.js?ver=5.2.3` |
| +0 |  |  | `www.ghacks.net` | 1 | script | `https://www.ghacks.net/wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1` |
| +0 |  |  | `www.ghacks.net` | 1 | script | `https://www.ghacks.net/wp-includes/js/jquery/jquery.js?ver=1.12.4-wp` |
| +0 | ##+js(abort-current-inline-script, jQuery, aba_detector) |  | `www.ghacks.net` |  | dom | `https://www.ghacks.net/` |
| +0 |  |  | `www.ghacks.net` | 1 | doc | `https://www.ghacks.net/` |
</details>
```

**Comment by gorhill** [comment]
I don't see any scriptlet injection filter for `ghacks.net` in default lists -- you stated the issue you reported occurs with default settings & lists. Where is that filter coming from?

**Comment by uBlock-user** [comment]
Extension console in Firefox is located at `about:devtools-toolbox?type=extension&id=uBlock0%40raymondhill.net` Just open that page and capture any errors or warnings appearing there.

**Comment by troysjanda** [comment]
@gorhill I am only using the deafult lists, forgot to put my personal list located here
`https://raw.githubusercontent.com/troysjanda/MyBlockLists/master/UBOXPI`

**Comment by troysjanda** [comment]
I removed FF and completely removed any leftovers and reinstalled FF after a reboot. Installed Ubo 1.22.3b16 and with just default filters checked seems to be working. I restored a ubo backup dated 09/10 and it seems to be ok. I then reinstalled all my normal addons, Nano defender, cookie auto delete, css exfil protection and Reddit and restored my personal filter lists and it seems to be working as well. sorry to drive you guys nuts. Hope that it remain fixed.
